scripts/in_container/check_environment.sh

```diff
@@ -112,10 +112,32 @@ function startairflow_if_requested() {
         echo
         export AIRFLOW__CORE__LOAD_EXAMPLES=${LOAD_EXAMPLES}
 
-        . "$( dirname "${BASH_SOURCE[0]}" )/configure_environment.sh"
+        if [ -z "${FILES_DIR+x}" ]; then
```

Comment:
I am not sure if in this case it's worth incorporating check environment in. What I really meant to is ratheer to rewrite the configure_environment.sh and run it from here. It makes little sense to increase duplication of the check_environment.sh into other scripts